app/src/main/java/org/permanent/permanent/ui/composeComponents/CustomDropdown.kt

```diff
@@ -23,21 +37,44 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.core.content.ContextCompat
+import kotlinx.coroutines.launch
 import org.permanent.permanent.R
+import org.permanent.permanent.models.ArchiveType
 
 @Composable
-fun CustomDropdown(isTablet: Boolean = false) {
+fun CustomDropdown(
+    isTablet: Boolean = false,
+    onArchiveTypeClick: (archiveType: ArchiveType) -> Unit
+) {
     val context = LocalContext.current
     val whiteColor = Color(ContextCompat.getColor(context, R.color.white))
     val purpleColor = Color(ContextCompat.getColor(context, R.color.barneyPurple))
     val accentColor = Color(ContextCompat.getColor(context, R.color.colorAccent))
+    val blue900Color = Color(ContextCompat.getColor(context, R.color.blue900))
+    val boldFont = FontFamily(Font(R.font.open_sans_bold_ttf))
     val semiboldFont = FontFamily(Font(R.font.open_sans_semibold_ttf))
     val regularFont = FontFamily(Font(R.font.open_sans_regular_ttf))
 
+    var currentArchiveType by remember {
+        mutableStateOf(
+            UIArchive(
+                ArchiveType.PERSON,
+                R.drawable.ic_heart_white,
+                R.string.personal,
+                R.string.personal_description
+            )
+        )
+    }
+
```

Comment:
var options = [UIArchive(
                ArchiveType.PERSON,
                R.drawable.ic_heart_white,
                R.string.personal,
                R.string.personal_description
            ),
UIArchive(
                ArchiveType.Family,
                R.drawable.ic_heart_white,
                R.string.personal,
                R.string.personal_description
            )]

options.forEach { it, index ->
Divider()
StorageMenuItem(
                        painterResource(id = it.icon),
                        stringResource(it.title),
                        stringResource(it.description)
                    ) {
                        currentArchive = it
                        onArchiveTypeClick(it.type)
                        scope
                            .launch { sheetState.hide() }
                            .invokeOnCompletion {
                                if (!sheetState.isVisible) {
                                    showBottomSheet = false
                                }
                            }
                    }
}
Divider()